main.py

```diff
@@ -1,7 +1,55 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from db.models import Genre, Actor
 
+genres = [
+    "Western",
+    "Action",
+    "Dramma"
+]
 
-def main() -> QuerySet:
-    pass
+actors = [
+    ("George", "Klooney"),
+    ("Kianu", "Reaves"),
+    ("Scarlett", "Keegan"),
+    ("Will", "Smith"),
+    ("Jaden", "Smith"),
+    ("Scarlett", "Johansson")
+]
+
+
+def main() -> None:
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for actor in actors:
+        for first_name, last_name in actor:
+            Actor.objects.create(
+                first_name=first_name[0],
+                last_name=last_name[1]
+            )
```

Comment:
It would be best if you unpacked `actors` as follows:
```suggestion
    for first_name, last_name in actors:
        Actor.objects.create(
            first_name=first_name,
            last_name=last_name
        )
```